<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QMainWindow" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="file_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_file">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>文件名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>大小</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>修改日期</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_file">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="enterButton">
           <property name="toolTip">
            <string>Enter</string>
           </property>
           <property name="text">
            <string>进入</string>
           </property>
           <property name="shortcut">
            <string>Return</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="process_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>进程名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>进程PID&amp;会话名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>会话数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>内存&amp;状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>cpu时间&amp;窗口标题</string>
          </property>
          <property name="textAlignment">
           <set>AlignJustify|AlignVCenter</set>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="refreshButton">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>刷新列表</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/actions/icons/refresh.png</normaloff>:/actions/icons/refresh.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="killButton">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>结束进程</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/actions/icons/killpocess.png</normaloff>:/actions/icons/killpocess.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action_sendFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/actions/icons/sendfile.png</normaloff>:/actions/icons/sendfile.png</iconset>
   </property>
   <property name="text">
    <string>传文件</string>
   </property>
   <property name="toolTip">
    <string>传文件</string>
   </property>
  </action>
  <action name="action_shutdown">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/actions/icons/shutdown.png</normaloff>:/actions/icons/shutdown.png</iconset>
   </property>
   <property name="text">
    <string>一键关机</string>
   </property>
   <property name="toolTip">
    <string>关闭对方计算机</string>
   </property>
  </action>
  <action name="action_process">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/actions/icons/process.png</normaloff>:/actions/icons/process.png</iconset>
   </property>
   <property name="text">
    <string>进程管理</string>
   </property>
   <property name="toolTip">
    <string>管理远程计算机进程</string>
   </property>
  </action>
  <action name="action_fullScreen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/actions/icons/fullscreen.png</normaloff>:/actions/icons/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>全屏显示</string>
   </property>
   <property name="toolTip">
    <string>最大化显示</string>
   </property>
  </action>
  <action name="action_unlock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/actions/icons/unlock.png</normaloff>:/actions/icons/unlock.png</iconset>
   </property>
   <property name="text">
    <string>解锁</string>
   </property>
   <property name="toolTip">
    <string>自动隐藏工具栏</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
